python/py-cryptography/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           openssl 1.0
 
 github.setup        pyca cryptography 35.0.0
 name                py-${github.project}
-revision            1
+revision            2
```

Comment:
Since you're only changing `py27-cryptography`, you shouldn't change the overall revision for the port.

Instead, you'll want to bump the revision specific for the Python 2.7 subport, on line 47. And it'll need to be changed to 3, as master is now at 2.